enterprise/reporting/error_extractor.go

```diff
@@ -109,48 +109,77 @@ func (ext *ExtractorHandle) getSimpleMessage(jsonStr string) string {
 	}
 
 	for key, erRes := range jsonMap {
-		erResStr, isString := erRes.(string)
+		if result := ext.handleKey(key, erRes); result != nil {
```

Comment:
Didn't get the motivation behind differentiating empty string vs no string. Was it a bug earlier?

Reply:
addressed